pynxtools_xps/config/config_specs_xy.json

```diff
@@ -58,7 +59,7 @@
         "associated_source":"@link:/entry/instrument/source_probe"
       },
       "ELECTRONANALYSER[electronanalyser]":{
-        "@default": "None",
+        "@default": "detector0",
```

Comment:
You replace the `DETECTOR[detector]` with `detector0` in your code, right? In this case this is fine, otherwise it needs to be named `detector`.

Reply:
Yes